packages/neuron-wallet/src/controllers/sudt.ts

```diff
@@ -0,0 +1,36 @@
+import LiveCellService from "services/live-cell-service";
+import env from 'env'
+import Script, {ScriptHashType} from "models/chain/script";
+import parseSUDTTokenInfo from "utils/parse_sudt_token_info";
+import {ResponseCode} from "utils/const";
+
+export interface TokenInfo {
+  tokenId: string
+  symbol: string
+  tokenName: string
+  decimal: string
+}
+
+export default class SUDTController {
+  private liveCellService = new LiveCellService()
+
+  public async getSUDTTokenInfo(params: { tokenId: string }): Promise<Controller.Response<TokenInfo>> {
+    let codeHash = process.env.TESTNET_SUDT_INFO_SCRIPT_CODEHASH!
+    if (!env.isTestMode) {
+      codeHash = process.env.MAINNET_SUDT_INFO_SCRIPT_CODEHASH!
+    }
```

Comment:
This would be better to be encapsulated in the existing `AssetAccountInfo` class, so it won't need to do this kind of environment check in the controller level.

Reply:
fix